apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { View, Text } from 'react-native';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+// const NotificationVariant string: 'primary' | 'neutral' | 'danger' | 'warning';
+enum NotificationVariant { primary = 1, neutral, danger, warning };
+
+const Notification = (props) => {
+  const variant = props.variant;
+  let startText = ''; 
+  let endText = '';
+
+  const styles: {[key: string]: any} = {
```

Comment:
I think it would be better if we used "StyleSheet.create()", so that we get proper type checking

https://reactnative.dev/docs/stylesheet